python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat_manager.py

```diff
@@ -157,7 +161,8 @@ async def handle_agent_response(self, message: GroupChatAgentResponse, ctx: Mess
                 return
 
             # Select a speaker to continue the conversation.
-            speaker_name_future = asyncio.ensure_future(self.select_speaker(self._message_thread))
+            messages = list(await self._message_store.get_messages())
```

Comment:
See the logic in SwarmGroupChatManager's implementation of select_speaker, it requires the most recent `HandoffMessage`.